packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -424,8 +509,20 @@ class FloatingActionButton extends StatelessWidget {
   ///  * [Feedback] for providing platform-specific feedback to certain actions.
   final bool? enableFeedback;
 
+
+  /// The spacing between the icon and the label for an extended
+  /// [FloatingActionButton].
+  ///
+  /// If this is null, the default is 8.0.
+  final double? extendedIconLabelSpacing;
+
   final BoxConstraints _sizeConstraints;
 
+  final _FloatingActionButtonType _floatingActionButtonType;
+
+  /// The label for an extended [FloatingActionButton].
```

Comment:
Don't need a public comment for a private field. Maybe just name this `extendedLabel` and remove the comment?

Reply:
Done